src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(ColoursDifferent.values().length);
```

Comment:
Consider renaming the enum `ColoursDifferent` to `Color` for better readability and simplicity. 😊